### PR TITLE
[SPARK-27336][SQL] Incorrect DataSet.summary() result

### PR DESCRIPTION
**Pull request by Le-Dai**
### What changes were proposed in this pull request?
Change the default value of accuracy

### Why are the changes needed?
I think it is necessary for the user to sacrifice memory to improve the accuracy of the result set.

### How was this patch tested?
retest the result set

### CODE REVIEW

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala [hunk 1]
```diff
@@ -51,7 +51,7 @@ import org.apache.spark.sql.types._
   usage = """
     _FUNC_(col, percentage [, accuracy]) - Returns the approximate percentile value of numeric
       column `col` at the given percentage. The value of percentage must be between 0.0
-      and 1.0. The `accuracy` parameter (default: 10000) is a positive numeric literal which
+      and 1.0. The `accuracy` parameter (default: 100000) is a positive numeric literal which
```
Comment (HyukjinKwon) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala]: I am not sure about this. The default is already documented and it's already able to configure. I wouldn't change this unless there's a strong reason to change its default.

Comment (MrDLontheway) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala]: how to configure

Comment (HyukjinKwon) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala]: You can set `accuracy` parameter

Comment (MrDLontheway) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala]: I found that I can change the parameters when I use, but summary() will call the default value. Can we configure the default value in the form of spark Config?

Comment (HyukjinKwon) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala]: It's just an approximate to summarize the current Dataset. If you need more accurate number, you can just manually call this function.

Comment (MrDLontheway) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentile.scala]: OK , thanks

**Pull request closed without merging**